### PR TITLE
Version 1.0.0

### PR DESCRIPTION
**Pull request by stevegrunwell**
This PR represents the initial release of the plugin, starting from scratch.

For reference, I've been working in `develop` since the beginning, but just created a new, empty `master` branch for the sake of being able to do a proper PR, then merged that unrelated history:

```sh
$ git checkout --orphan master
$ git commit --allow-empty -m "Initial commit"
$ git checkout develop
$ git merge master --allow-unrelated-histories
```

### CODE REVIEW

File: src/OrderLimiter.php [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+				if ( $current_dow !== $start_of_week ) {
+					if ( $current_dow > $start_of_week ) {
+						$diff = $current_dow - $start_of_week;
+					} elseif ( $current_dow < $start_of_week ) {
+						$diff = $current_dow + 7 - $start_of_week;
+					}
+
+					$start = $start->sub( new \DateInterval( 'P' . $diff . 'D' ) );
+				}
+				break;
```
Comment (bswatson) [inline src/OrderLimiter.php]:
```suggestion
				$diff = $current_dow - $start_of_week;
				// Adjust the diff if the start of the week is numerically higher than the current day. 
				if ( $diff < 0 ) {
					$diff = $diff + 7;
				}
				
				if ( $diff > 0 ) {
					$start = $start->sub( new \DateInterval( 'P' . $diff . 'D' ) );
				}		
```

Comment (bswatson) [reply src/OrderLimiter.php]: No change needed here, but was trying work through the logic and made a simplified version in the process.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Changelog
```
Comment (bswatson) [inline CHANGELOG.md]: @stevegrunwell Do we need to update this before merging to master?

Comment (stevegrunwell) [reply CHANGELOG.md]: Indeed, will get that taken care of ASAP

File: readme.txt [hunk 3]
```diff
@@ -0,0 +1,49 @@
+=== Limit Orders for WooCommerce ===
+Contributors: nexcess, liquidweb, stevegrunwell
+Tags: WooCommerce, ordering, limits, throttle
+Requires at least: 5.3
+Tested up to: 5.4
+Requires PHP: 7.0
+Stable tag: 0.1.0
```
Comment (bswatson) [inline readme.txt]: Are we still considering this a `0.1.0` version, or would this be 1.0.0?

Comment (stevegrunwell) [reply readme.txt]: I suppose since we're putting it out there (and it has a plugin banner and everything), I'll go ahead and bump to 1.0.0 :wink:

**Pull request merged**